Sources/StreamChatUI/CommonViews/PillButton.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class PillButton: MediaButton {
```

Comment:
Especially this one should be documented since it is not clear at first sight what this button is.